SEP-0002-revised.md

```diff
@@ -0,0 +1,245 @@
+Candidates which have accepted their nomination (self nominations are allowed) should write a short statement which will be made public as soon as possible, but no later than before the election starts.
+* Election: All candidates that have written a short statement will be put to a vote of the project members and the candidate with the most votes shall be elected to the position.
+In the event that more than one position has to be filled simultaneously each election shall be treated as a separate election for the purposes of this process.
+
+### Term
+
+Each position on the steering committee has a term of three years, from the date of the announcement of the election results.
+
+If there is a vacancy part way through a term, then a by-election will be held where the candidate elected to the committee will serve the remainder of the term of the member they are replacing.
+
+For the first election of the coordinating committee, one member will have a one year term, one member a two year term and one member a three year term.
+The term length of the elected candidates may be decided between themselves, in the case agreement can not be reached the person elected to the financial officer role will serve three years, the deputy lead developer two years and the lead developer one year.
+
+### Removing Members
+
+In exceptional circumstances, the project members may remove one or more sitting steering committee members via a single vote.
+
+Such a vote is triggered when a project member calls for one publicly on an appropriate Project-wide communication channel, and two other active project members second the proposal.
+
+For Steering Committee Member(s) to be removed, two-thirds of the active Project Members (where the count includes the member being voted on if they are a project member) must vote in support of the removal.
+
+To avoid repeated removal votes, any individual Project Member who has called for, or seconded such a vote, may not call for or second a vote to remove any Steering Committee member for one year from the original motion.
+
+## The Project Members
+
+The project members are trusted community members with a proven track record of constructive contribution to the goals of the project.
+Collectively, the project members make decisions about the direction of the project and have authority over the project, which is delegated to the Steering Committee.
+
+Project Members are expected to act as role models for the community and custodians of the SunPy Project, on behalf of the community and all those who rely on SunPy.
+They will act as representatives of the SunPy Project, where necessary, including in online discussions or at official SunPy events.
+
+### Responsibilities and Rights
+
+Project Members may and should participate in formal votes on:
+
+* Adding new Project Members
+* Electing Steering Committee members
+* Appointment of the Ombudsperson
```

Comment:
the board nominates the ombudsperson and the project members approves? is that right?

Reply:
yes, that is the intent

Reply:
See the section below on the ombudsperson.